assemblies/assembly-planner-configuration.adoc

```diff
@@ -0,0 +1,1986 @@
+
+//:OPTAPLANNER-COMM:
+//Variable for the community version conditional statements.
+:OPTAPLANNER-ENT:
+//Variable for the enterprise version conditional statements.
+
+////
+ifdef::OPTAPLANNER-COMM[]
+[[plannerConfiguration]]
+= OptaPlanner configuration
+:doctype: book
+:imagesdir: ..
+:sectnums:
+:toc: left
+:icons: font
+:experimental:
+
+
+[[plannerConfigurationOverview]]
+== Overview
+endif::OPTAPLANNER-COMM[]
+////
+
+
+
+[id='assembly-planner-configuration_{context}']
+
+= The {PLANNER} solver
+
+Solving a planning problem with OptaPlanner consists of the following steps:
+
+. *Model your planning problem* as a class annotated with the `@PlanningSolution` annotation, for example the `NQueens` class.
+. *Configure a Solver*, for example a First Fit and Tabu Search solver for any `NQueens` instance.
+. *Load a problem data set* from your data layer, for example a Four Queens instance. That is the planning problem.
```

Comment:
The description is based on NQueens example, the picture under is for cloud balancing and our standard hello world is for time tabling. Maybe it is worth to keep only one example. I see that in community doc we have the same issue. However that maybe would be better if we can use one only example here

Reply:
However do not relate to this PR feel free to ignore

Reply:
@dupliaka I agree. I was trying to keep this as close as possible to the community doc. However, to change all of the examples to timetable will take some effort. I will create a separate jira for that. WDYT? This won't be published until 7.12 so we have time.

Reply:
Yes, let's create a separate jira for that